apps/springboard-test/index.ts

```diff
@@ -0,0 +1,31 @@
+import {Springboard} from 'springboard/engine/engine';
+import {springboard} from 'springboard/engine/register';
+
+const mod = springboard.defineModule('Main', {
```

Comment:
We should then be able to do the following, even though it wouldn't make sense for a Main module:

```ts
declare module 'springboard' {
    interface DefinedModules {
        Main: typeof mod;
    }
}
```